### PR TITLE
SVP - Change logic on medication & Add Red Color on Immunisation PastDue date

### PR DESCRIPTION
**Pull request by Diama1**

### CODE REVIEW

File: client/src/elm/Pages/WellChildActivity/View.elm [hunk 1]
```diff
@@ -1139,7 +1146,7 @@ viewVaccinationOverview language currentDate child vaccinationProgress db =
                     |> List.sortWith Date.compare
                     |> List.map (formatDDMMYYYY >> text >> List.singleton >> p [])
                     |> div [ class "cell date" ]
-                , div [ class "cell next-due" ]
+                , div [ class <| "cell next-due " ++ nextDueClass ]
```
Comment (anvmn) [inline client/src/elm/Pages/WellChildActivity/View.elm]: Use `classList`

**Review by anvmn — CHANGES_REQUESTED** [review]
@Diama1 

One comment  + add a screenshot please.

**Comment by Diama1** [comment]
Screen Shot 2021-12-17 at 13 11 18
Screen Shot 2021-12-16 at 14 43 09
Screen Shot 2021-12-16 at 14 44 51

**Review by anvmn — APPROVED** [review]
Thanks @Diama1

**Pull request merged**